T5/src/Environment/context/Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM mcr.microsoft.com/azureml/curated/acpt-pytorch-2.0-cuda11.7
 
-RUN pip install azureml-evaluate-mlflow
-RUN pip install git+https://github.com/huggingface/accelerate
-RUN pip install evaluate datasets rouge-score nltk py7zr optimum
-RUN pip install git+https://github.com/huggingface/transformers
-RUN pip install  git+https://github.com/huggingface/peft.git
-RUN pip install diffusers==0.16.1
\ No newline at end of file
+RUN pip install azureml-evaluate-mlflow==0.0.14.post1
```

Comment:
From a maintainability standpoint, do we need to periodically update the package versions in this dockerfile?

Reply:
Since there isn't a direct customer yet, I think its better to keep the versions tight so it always works for anyone who comes across the example. If we end up having a scenario where we need to actively support T5, we can upgrade the versions.